config/fxdata/magic.cfg

```diff
@@ -1329,6 +1341,28 @@ WithstandHitAgainst = WALL DOOR
 Properties = NO_HIT REBOUND_IMMUNE WIND_IMMUNE EXPLODE_FLESH NO_STUN
 UpdateLogic = 7
 
+[shot33]
+Name = SHOT_RANGED_HEAL
+Animation = 0
+AnimationSize = 0
+AnimationTransparency = 0
+Size_XY = 128
+Size_Z = 128
+Health = 2
+Damage = 0
+DamageType =
+HitType = 11
+Speed = 256
+MaxRange = 5120
+BaseExperienceGain = 0
+DestroyOnHit = 0
+TargetHitstopTurns = 0
+SpellEffect = 32
+Properties = REBOUND_IMMUNE WIND_IMMUNE HIDDEN_PROJECTILE
+FireLogic = 1
+EffectModel = EFFECTELEMENT_RED_DOT
```

Comment:
Currently you have drain as a placeholder. We should change this into a different effect element so it looks better. I did a suggestion on discord, but if you want to design something yourself or want me to change something I would be more than happy with that too.

Reply:
Done.